suite/performance/basic_test.go

```diff
@@ -5,53 +5,55 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/geth"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	uuid "github.com/satori/go.uuid"
 	"github.com/smartcontractkit/chainlink-testing-framework/actions"
 	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 	"github.com/smartcontractkit/chainlink-testing-framework/client"
-	"github.com/smartcontractkit/chainlink-testing-framework/config"
 	"github.com/smartcontractkit/chainlink-testing-framework/testsetups"
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
-	"github.com/smartcontractkit/helmenv/environment"
 )
 
 var _ = Describe("Profiling suite @profile", func() {
 	var (
-		err             error
-		nets            *blockchain.Networks
-		chainlinkNodes  []client.Chainlink
-		mockserver      *client.MockserverClient
-		testEnvironment *environment.Environment
-		profileTest     *testsetups.ChainlinkProfileTest
+		err            error
+		nets           *blockchain.Networks
+		chainlinkNodes []client.Chainlink
+		ms             *client.MockserverClient
+		e              *environment.Environment
+		profileTest    *testsetups.ChainlinkProfileTest
 	)
 
 	BeforeEach(func() {
 		By("Deploying the environment", func() {
-			testEnvironment, err = environment.DeployOrLoadEnvironment(
-				environment.NewChainlinkConfig(
-					config.ChainlinkVals(),
-					"chainlink-profiling",
-					config.GethNetworks()...,
-				),
-			)
-			Expect(err).ShouldNot(HaveOccurred(), "Environment deployment shouldn't fail")
-			err = testEnvironment.ConnectAll()
+			e = environment.New(nil)
+			err := e.
+				AddHelm(mockservercfg.New(nil)).
```

Comment:
`AddDeployment()` instead of `AddHelm()`? K8s has proven to be enough of a stumbling block for most outside users, would rather not imply they need to know about helm as well.

Reply:
Looking to simplify that, but for now, it's better to explicitly state which kind of deployment we are using, `chainlink-env` have multiple.